config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
yeah, I didn't realize this won't help because this is before model calls `audited` method which sets it to [true](https://github.com/collectiveidea/audited/blob/v3.0.0/lib/audited/auditor.rb#L90)